index.html

```diff
@@ -84,6 +84,36 @@
 
 				<div class="border1"></div>
 
+                <div class="card-content white-text">
+	                <span class="card-title">Share Game</span>
+				</div>
+				<div class="card-action">
+					<ul class="social-icons">
+						<div class="social-container">
```

Comment:
here, you can switch ul and li
There is no problem in the section below having a div in the ul.
It should look like this;

```
<div> 
  <ul>
       no div here
    <li>
      <div>
```

Reply:
![image](https://user-images.githubusercontent.com/34100642/34880074-d6d50676-f7e1-11e7-9d8b-d7c42d5b25d9.png)
I am confused for this part? Did < li > 94 move to line 92?

Reply:
No.Change div inside the `ul` to `li`.  you can use li within another li